pkg/buildpack/pod.go

```diff
@@ -26,19 +26,15 @@ func WaitAndGetPod(c *kubernetes.Clientset, application types.Application) (*cor
 	}
 	defer w.Stop()
 
-	const timeoutInSeconds = 10
+	const timeoutInSeconds = 30
 	duration := timeoutInSeconds * time.Second
 	select {
 	case val := <-w.ResultChan():
+		log.Debugf("Received event of type %s", val.Type)
 		if e, ok := val.Object.(*corev1.Pod); ok {
-			log.Debugf("Status of %s pod is %s", e.Name, e.Status.Phase)
-			switch e.Status.Phase {
-			case corev1.PodRunning:
-				log.Debugf("Pod %s is running.", e.Name)
-				return e, nil
-			case corev1.PodFailed, corev1.PodUnknown:
-				return nil, errors.Errorf("pod %s status %s", e.Name, e.Status.Phase)
-			}
+			return e, nil
```

Comment:
We should name that variable something else. Same for `ok`. This is confusing and goes pretty much against Go idioms.

Reply:
I'll update it

Reply:
`ok` is a go idiom isn't it?

Reply:
I'm actually not familiar with this one… usually expecting `whatever, error`, not `something, ok` but in this case, it's not like we have a choice anyway…

Reply:
[This](https://golang.org/doc/effective_go.html#interface_conversions) implies that when a boolean instead of an error type, `ok` is the way to go

Reply:
Nice, didn't know that. Thank you!

Reply:
We're learning the wonderful world of go 😎